tox.ini

```diff
@@ -13,6 +10,14 @@ commands =
    python -V
    nosetests []
 
+[testenv:docs]
+commands =
+   python setup.py build_sphinx
+
+[testenv:update-pep8]
+commands =
+   {toxinidir}/scripts/update-pep8.sh
```

Comment:
yes, if you see the update-pep8.sh, we run "git status" and fail if any files need updates